bindings/protocols/bacnet/bacnet-model.ttl

```diff
@@ -53,33 +53,50 @@ Additional parameters can be specified via URI variable syntax, e.g. ?bacnetCmdP
 
 bacnet:usesService rdf:type owl:DatatypeProperty ;
                    rdfs:domain hctl:Form ;
-                   rdfs:range [owl:oneOf ("ReadProperty"^^xsd:string "WriteProperty"^^xsd:string "SubscribeCOV"^^xsd:string "UnsubscribeCOV"^^xsd:string)] ;
+                   rdfs:range [owl:oneOf ("ReadProperty"^^xsd:string "WriteProperty"^^xsd:string "SubscribeCOV"^^xsd:string)] ;
                    rdfs:comment """
 Default BACnet service by td:OperationType:
 - td:readProperty -> ReadProperty
 - td:writeProperty -> WriteProperty
 - td:observeProperty -> SubscribeCOV, subscribe to object if no property ID is present in target, otherwise subscribe to property
-- td:unobserveProperty -> UnsubscribeCOV
+- td:unobserveProperty -> SubscribeCOV (will do an unsubscribe instead)
 
 TODO: Can we specify the defaults in a machine-readable, RDF-standard way?
 
 Not mapped yet:
-- td:invokeAction
+- td:invokeAction, td:queryAction, td:cancelaction
```

Comment:
should it be camel case, i.e. cancelAction?

I didn't know these were in the TD namespace yet

Reply:
I was not sure, just followed the camel case convention. 😄 

But I doublechecked now, these are defined like this in JSON-LD context <https://www.w3.org/2022/wot/td/v1.1> and in the TD ontology: <https://www.w3.org/2019/wot/td#invokeAction> So we should be good here.

Reply:
td:cancelAction should be camel case, I will correct

Reply:
For the record: the TD instances have all lower case, ontology is camel case with the translation done in JSON-LD Context.

Reply:
I fixed them now.